optax/transforms/_constraining_test.py

```diff
@@ -114,6 +114,15 @@ def test_zero_nans(self):
             (jnp.array(False), jnp.array(False), jnp.array(False))))
     chex.assert_trees_all_close(updates, grads)
 
+  def check_none_arguments(self):
```

Comment:
call it `test_none_arguments`

Reply:
Also why are you testing for this to raise an error?

Reply:
Done. In this function is a check that raises a ValueError if None is passed as params. Hence, the tree error can only occur if this check is, for some reason, disabled. If we rely on this check not being modified, we can also remove this test case entirely.

Reply:
Right, perfect, thank you for the explanation!